src/main/java/software/amazon/awssdk/crt/http/HttpStream.java

```diff
@@ -55,6 +63,57 @@ public void incrementWindow(int windowSize) {
         }
     }
 
+    /**
+     * Use only for Http 1.1 Chunked Encoding. At some later point we may adapt this interface for H2, but not yet. If
```

Comment:
How would h2 be any different?

Reply:
If you were sending something over h2 for a use case you'd normally use chunked encoding for in http 1.1, the API should be identical. But, I don't think it's plumbed together in aws-c-http at the moment. When it is, I'll connect the pieces here and change the comment.